src/components/GGanttBar.vue

```diff
@@ -106,18 +114,6 @@ onMounted(() => {
     { deep: true, immediate: true }
   )
 })
-
-const barStyle = computed(() => {
-  return {
-    ...barConfig.value.style,
-    position: "absolute",
-    top: `${rowHeight.value * 0.1}px`,
-    left: `${xStart.value}px`,
-    width: `${xEnd.value - xStart.value}px`,
-    height: `${rowHeight.value * 0.8}px`,
-    zIndex: isDragging.value ? 3 : 2
-  } as CSSProperties
```

Comment:
Just saw that I still need to remove the CSSProperties type import, since I don't use that anymore.

Reply:
Fixed ✅